src/test/collaterals.sol

```diff
@@ -0,0 +1,1528 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.12;
+
+contract Collaterals {
+    struct SystemValues {
```

Comment:
Same comment as at https://github.com/makerdao/spells-goerli/pull/37#pullrequestreview-848566263

We probably want to move the system values out of the collaterals contract or just call this something more generic.

Reply:
i'll make it something more generic, i need the system values in there so i can set the collaterals to it otherwise i was running into a stack too deep problem